src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -3112,6 +3112,116 @@ public void NoWarn_SpecificNoWarnOverridesSpecificWarnAsError()
             Assert.Equal(expected: ReportDiagnostic.Suppress, actual: arguments.CompilationOptions.SpecificDiagnosticOptions["Test001"]);
         }
 
+        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

Consider adding a test for `/warnAsErrors:nullable`, regardless of whether we decide to support it or not.

Reply:
See `CommandLineTests.WarnAsError_Nullable`